rpc/src/v1/types/transaction_with_hash.rs

```diff
@@ -15,7 +15,9 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 use ckey::{NetworkId, PlatformAddress};
-use ctypes::transaction::{AssetMintOutput, AssetTransferInput, AssetTransferOutput, Transaction as TransactionType};
+use ctypes::transaction::{
+    AssetMintOutput, AssetTransferInput, AssetTransferOutput, OrderOnTransfer, Transaction as TransactionType,
```

Comment:
You should use `OrderOnTransfer` defined in the rpc module.

Reply:
Fixed.